src/libsystemd-network/sd-ndisc-router.c

```diff
@@ -5,12 +5,15 @@
 
 #include <netinet/icmp6.h>
 
+#include "dns-resolver-internal.h"
 #include "sd-ndisc.h"
```

Comment:
we want that `sd-*.h` includes are placed first, and all others aferwards, separated by empty line.

i.e. sd-*.h includes are public interfaces, and by ordering them first we want to validate they work without any internal definitions pulled in.

Reply:
Oh, clever. Thanks for the explanation.